pkgs/build-support/cc-wrapper/default.nix

```diff
@@ -40,7 +40,7 @@ let
   libc_lib = if libc == null then null else getLib libc;
   cc_solib = getLib cc;
   # The wrapper scripts use 'cat' and 'grep', so we may need coreutils.
-  coreutils_bin = if nativeTools then "" else getBin coreutils;
+  coreutils_bin = if (nativeTools || targetPlatform.isWindows) then "" else getBin coreutils;
```

Comment:
This `coreutils` for the build platform, not host platform, so why disable it?

Reply:
This ends up trying to build `coreutils` for `mingw32` when cross compiling `gcc`.
And `coreutils` can't be built on `mingw32`.